lib/meadow/data/work.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Meadow.Data.Work do
+  @moduledoc """
+  A repository data object. Embeds one Metadata map and many FileSets.
+  """
+
+  use Ecto.Schema
+  alias Meadow.Data.{FileSet, Metadata}
+
+  import Ecto.Changeset
+
+  @primary_key {:id, Ecto.ULID, autogenerate: true}
+  schema "works" do
+    field :visibility, :string
+    field :work_type, :string
+
+    embeds_one :metadata, Metadata, on_replace: :update
+    embeds_many :file_sets, FileSet, on_replace: :delete
+
+    timestamps()
+  end
+
+  def changeset(work, attrs) do
+    work
+    |> cast(attrs, [:work_type, :visibility, :metadata, :file_sets])
+    |> validate_required([:work_type, :visibility, :metadata])
```

Comment:
I'm wondering about this validate_required. Is it requiring title? When the skeleton works are created, they don't have titles yet.